stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -1030,9 +1033,67 @@ void SetUpInterfacesInterfaceEthernetConfigMacAddress(uint64 mac_address,
                         stream);
   };
   auto on_change_functor = UnsupportedFunc();
+
+  auto on_set_functor =
+      [node_id, port_id, node, tree](
+          const ::gnmi::Path& path,
+          const ::google::protobuf::Message& val,
+          CopyOnWriteChassisConfig* config) -> ::util::Status {
+    const gnmi::TypedValue* typed_val =
+        dynamic_cast<const gnmi::TypedValue*>(&val);
+    if (typed_val == nullptr) {
+      return MAKE_ERROR(ERR_INVALID_PARAM) << "not a TypedValue message!";
+    }
+    std::string mac_address_string = typed_val->string_val();
+    ::google::protobuf::uint64 mac_address =
+        YangStringToMacAddress(mac_address_string);
+    if (mac_address_string.compare("00:00:00:00:00:00") !=0 && mac_address == 0) {
+      return MAKE_ERROR(ERR_INVALID_PARAM) << "wrong value!";
+    }
+
+    // Set the value.
+    auto status = SetValue(node_id, port_id, tree,
+                           &SetRequest::Request::Port::mutable_mac_address,
+                           &MacAddress::set_mac_address, mac_address);
+    if (status != ::util::OkStatus()) {
+      return status;
+    }
+
+    // Update the chassis config
+    ChassisConfig* new_config = config->writable();
```

Comment:
Minor nitpick: Do you see an easy way to hit this code in a test?
Would improve test coverage and make the coverall check happy.

Reply:
The issue: Fix ChassisConfig in YangParseTreeTest reported by Yi Tseng results in this low coverage problem.

Reply:
Ok then. 80% should work too.